src/controllers/projectController.ts

```diff
@@ -25,10 +26,36 @@ export class ProjectController implements Disposable {
     }
 
     public async createJavaProject() {
-        const javaVersion: number = await this.getJavaVersion();
-        if (!javaVersion) {
-            return;
+        const projectKinds: QuickPickItem[] = [{
+            label: BuildTool.None,
+            detail: "A project without any build tools",
+        }];
+        if (contextManager.getContextValue(Context.MAVEN_ENABLED)) {
+            projectKinds.push({
+                label: BuildTool.Maven,
+                detail: "Use Maven as the build tool of your Java project",
```

Comment:
> Apache Maven is a software project management and comprehension tool. Based on the concept of a project object model (POM), Maven can manage a project's build, reporting and documentation from a central piece of information.

so would something like "Use Maven to manage your project" be simple and concise?